### DISCUSSION TITLE
Gestion Git

### DISCUSSION THREAD

**Discussion by amrx12** [lucile98-ode/GPI-L2BHA-FENOUIL#36] [Show and tell]
**N'oubliez pas de faire un pull request pour les branches qui n'ont pas étés raccordées à la branche main (si cela est nécessaire bien sur) Mais pensez que le rendu final doit être présent sur main**

**Comment by lucile98-ode** [comment]
je vais faire ça après la création des diapos

**Reply by HaceneM** [reply]
Les branches non raccordés ont été pull
